### PR TITLE
Sarwara2

### PR DESCRIPTION
**Pull request by sarwarahmed1**

### CODE REVIEW

**Review by Mariachaudhry — APPROVED** [review]

**Review by zamant2 — APPROVED** [review]

File: src/test/java/ServiceNow/ESR/Pages/NCINativeViewPage.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    public NCINativeViewPage() {
+        PageFactory.initElements(WebDriverUtils.webDriver, this);
+    }
+
+    /** Filter navigator in Native View NCI Home Page */
+    @FindBy(xpath = "//input[@placeholder='Filter navigator']")
+    public WebElement nativeViewFilterNavigator;
+
+    /** Create ESR Button under the Engineering Service Request section */
+    @FindBy(xpath = "//ul[@aria-label='Modules for Application: ESR - Engineering Support Request']//li[6]//div[1]//div[1]//a[1]")
```
Comment (hamidnesar) [inline src/test/java/ServiceNow/ESR/Pages/NCINativeViewPage.java:20]: I would suggest using dynamic xpath over indexing as slight UI change might impact it.

**Review by hamidnesar — APPROVED** [review]
I did not see any screenshot being captured in the implemented steps. I am not sure if they were not needed but it would add great visibility to the report.

**Review by alenaminsk — APPROVED** [review]

**Review by mzuhoor — APPROVED** [review]

**Review by iamdez99 — APPROVED** [review]
Reviewed and approved - juarezds

**Review by iamdez99 — APPROVED** [review]
Reviewed and approved - juarezds

**Pull request merged**